cmd/controller/main.go

```diff
@@ -110,6 +81,8 @@ func startPkgController(stopCh <-chan struct{}, cfg *rest.Config, logger *zap.Su
 	brokerInformer := eventingInformerFactory.Eventing().V1alpha1().Brokers()
 	coreServiceInformer := kubeInformerFactory.Core().V1().Services()
 	coreNamespaceInformer := kubeInformerFactory.Core().V1().Namespaces()
+	serviceInformer := kubeInformerFactory.Core().V1().Services()
```

Comment:
there are two informers defined for services. coreServiceInformer and serviceInformer. coreServiceInformer was a copy from serving where they also have a service. 

I think the right answer is to rename both of the core things.

```
	coreServiceInformer := kubeInformerFactory.Core().V1().Services()
	coreNamespaceInformer := kubeInformerFactory.Core().V1().Namespaces()
```

becomes 

```
	serviceInformer := kubeInformerFactory.Core().V1().Services()
	namespaceInformer := kubeInformerFactory.Core().V1().Namespaces()
```

And then updates where needed.

Reply:
Done!